app/models/user_annotation.rb

```diff
@@ -14,12 +14,39 @@ class UserAnnotation
   ###
 
   include Mongoid::Document
+  include Swagger::Blocks
+
   field :name, type: String
   field :values, type: Array
   field :queued_for_deletion, type: Boolean, default: false
   field :publishing, type: Boolean, default: false
   field :source_resolution, type: Integer
 
+  # Define parameters used in API POST endpoint to create user annotations
+  swagger_schema :UserAnnotationInput do
+    key :name, 'UserAnnotation'
+    property :name do
+      key :type, :string
+      key :description, 'Name of new custom user annotation'
+    end
+    property :user_data_arrays_attributes do
+      key :type, :object
+      key :description, 'Labels (names) and cell name arrays (values) for each selection'
+    end
+    property :cluster do
+      key :type, :string
+      key :description, 'Cluster group name'
+    end
+    property :subsample_threshold do
+      key :type, :string
+      key :description, 'Subsample threshold'
+    end
+    property :subsample_annotation do
+      key :type, :string
+      key :description, 'Subsample annotation'
+    end
+  end
```

Comment:
we discussed moving this into the controller, since API docs don't really belong in models -- is that still true, or are there other concerns?

Reply:
I had qualms after I looked back, and saw I put this in a model because that's where all 26 other SCP `swagger_schema` declarations are.  But I chatted with Jon, and he agrees with your rationale, so I've made your suggested change.